Monika After Story/game/script-topics.rpy

```diff
@@ -901,30 +935,46 @@ label monika_okayeveryone:
     m 5hua "Ahaha~"
     return
 
-
+# TODO needs update script to lock for clearall persistents. Since this is assuming the others are dead, doesn't make sense for a clearall playthru
 init 5 python:
-    addEvent(Event(persistent.event_database,eventlabel="monika_whispers",category=['ddlc','club members'],prompt="Others still lingering",random=True))
+    addEvent(
```

Comment:
for this, instead of having a conditinal, why dont we just enable/disable the event object based on the persistent value. so like this:
```suggestion
    if not persistent.clearall:
        addEvent(...
        
```

conditional property should be used for events that should unlock/do something after something has been reached, not perma-unlock/lock based on a starting condition. This also means the update script would not be needed